### ISSUE TITLE
iedit fails when both gtags and git active

### ISSUE THREAD

**Issue by smallid** [syl20bnr/spacemacs#8885]
#### Description :octocat:
iedit fails when both gtags and git active

#### Reproduction guide :beetle:
- create new git repo 'test' `git init test`
- create new file test.cpp with "test test" `echo "test test" > test/test.cpp`
- run gtags `cd test; gtags`
- Start Emacs
- open test.cpp
- move cursor over 'e' in first "test", enter iedit mode (SPC s e)
- type 'i' for insert mode, then type character 'i'

*Observed behaviour:* :eyes: :broken_heart:
Second copy of "test" not modified, resulting in `tiest test`

*Expected behaviour:* :heart: :smile:
`tiest tiest`

#### System Info :computer:
- OS: gnu/linux
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: master (rev. 8e1af145)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(helm better-defaults emacs-lisp c-c++ git markdown gtags themes-megapack)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS NOTIFY ACL GNUTLS LIBXML2 FREETYPE XFT ZLIB TOOLKIT_SCROLL_BARS GTK2 X11

**Comment by smallid** [comment]
Disabling gtags highlighting, then reloading the file seems to avoid this issue: `(setq ggtags-highlight-tag nil)`

May be related to #8527

**Comment by cserb** [comment]
I can confirm the same behaviour and also that turning off gtags highlighting fixes the issue. Thanks @smallid

**Comment by smallid** [comment]
Yes, it is still valid Mr bot. Not fixing a problem does not make it go away.
